pcsx2/Achievements.cpp

```diff
@@ -1983,7 +1981,7 @@ void Achievements::DrawGameOverlays()
 
 			TinyString width_string;
 			width_string.append(ICON_FA_STOPWATCH);
-			for (u32 i = 0; i < indicator.display_length; i++)
+			for (u32 i = 0; i < indicator.text.length(); i++)
 				width_string.append('0');
```

Comment:
```suggestion
			width_string(indicator.text.length(), '0');
```

Reply:
It's not a std::string. Probably should add that to TinyString though.